README.md

```diff
@@ -1,2 +1,89 @@
-# Hello-World-Project
-This is the first Microverse Project meant to test application of gitflow and es linter configuration
+📗 Table of Contents
+
+    📖 About the Project
+        🛠 Built With
+            Tech Stack
+            Key Features
+        🚀 Live Demo
+    💻 Getting Started
+        Setup
+        Prerequisites
+        Install
```

Comment:
You have done a great job by utilizing the readme template as suggested by the previous reviewer. However, the readme seems not to render properly because the markdown is not as it's on the template.

Kindly follow the illustration below to copy the raw content of the template from [here](https://github.com/microverseinc/readme-template/blob/master/README.md):
![image](https://user-images.githubusercontent.com/20554583/204653999-ba3c6714-1f58-490e-b83b-4006607bef0d.png)

How it renders currently:

![image](https://user-images.githubusercontent.com/20554583/204654310-5eb37f0a-2f28-40c1-8d09-4b50139d7318.png)


How it is supposed to render:
![image](https://user-images.githubusercontent.com/20554583/204654292-99739c0c-fa80-49b6-bc1d-35473bf90550.png)


Kindly implement this and your project will be ready for approval.